packages/coinstac-ui/app/render/components/dashboard-consortia.js

```diff
@@ -23,12 +23,12 @@ class DashboardConsortia extends Component {
 
           return (
             <ConsortiumCard
-              deleteConsortium={() => dispatch(deleteConsortium(id))}
```

Comment:
In cases where you’re using multiple properties/functions from `this.props` it’s nice to destructure:

```js
const {
  deleteConsortium,
  joinConsortium,
  leaveConsortium,
} = this.props;

// ...
 <ConsortiumCard
  deleteConsortium={() => deleteConsortium(id)}
  joinConsortium={() => joinConsortium(id, username)}
  leaveConsortium={() => leaveConsortium(id, username)}
/>
```

A little nicer to read.